1-Draft/RFC00XX-Update-Error-View.md

```diff
@@ -0,0 +1,226 @@
+---
+RFC: Update $ErrorView with simplified views for Error Messages
+Author: Jason Helmick
+Status: Draft
+Version: 0.0.0
+Area: PowerShell
+Comments Due: 10/31/2019
+---
+
+# Update $ErrorView with simplified views for Error Messages
+
+When an error occurs in PowerShell, the customers on-screen error message experience currently
+provides a level of detail that obscures the exception message from being recognized and read by new
+or occasional PowerShell users. The addition of a simplified error view will improve both
+comprehension and troubleshooting experience. A new cmdlet `Get-Error` will provide
+complete detailed view of the fully qualified error when desired.
+
+## Motivation
+
+The on-screen experience, when receiving an error message, is controlled through the views
+NormalView (the default) and CategoryView. These are user selectable through the preference variable
+**$ErrorView**. This RFC describes Changing **$ErrorView** to an enumeration and adding one
+additional dynamic view to improve readability; 'ConciseView'
+
+- ConciseView - provides a concise error message suitable for new or occasional PowerShell users and
+  a refactored view for advanced module builders. If the error is not from a script or parser error,
+  then it's a single line error message. Otherwise, you get a multiline error message that contains
+  the error, a pointer and error message showing where the error is in that line. If the terminal
+  doesn't support Virtual Terminal, then vt100 color codes are not used.
+
+A comprehensive detailed view of the fully qualified error, including inner exceptions, will be
+provided by the `Get-Error` cmdlet.
+
+**$ErrorView** shall contain the original views for backward compatibility and to lessen this
+breaking change. The view list is as follows:
+
+- ConciseView
+- NormalView
+- CategoryView
+
+## Specification
+
+The proposal is to add one new view to help improve error message comprehension based on user needs.
```

Comment:
We should make clear that the intent is for `ConciseView` to be the default

Reply:
This has been updated in the RFC - Thanks!